src/pages/Home.tsx

```diff
@@ -1,14 +1,64 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+import { getPosts } from '../store/posts/action';
+import { Posts } from '../store/posts/types';
+import PostCard from '../components/PostCard/PostCard';
+import { SearchBar } from '../components/SearchBar/SearchBar';
+import { RootState } from '../store/rootReducer';
+
+type PostCard = {
```

Comment:
If you are not using this type, you can delete it.
Also you have already written it in PostCard.tsx